org-formation/300-account-defaults/organization-tasks.yaml

```diff
@@ -59,3 +59,20 @@ SceptreLambdaBucket:
     IncludeMasterAccount: true
     Account: '*'
     Region: us-east-1
+
+ItKmsKey:
+  Type: update-stacks
+  Template: ./kms-key.yaml
+  StackName: it-kms-key
+  MaxConcurrentStacks: 10
+  Parameters:
+    AliasName: alias/itkey
```

Comment:
This name is a little too nonspecific... Maybe use `alias/it-buildtin-key` so that it can be differentiated from `alias/it-specialproj-key`?

Reply:
well, i don't think we would call anything `it-specialproj-key`, each project might have a key of their own and it would probably just be named `specialproj-key`.  how name it `it-infra`?

Reply:
alias name updated